.github/workflows/agentic-tests.yml

```
@@ -0,0 +1,246 @@
+
+            After reading and adopting that persona, respond to this prompt:
+            ${{ matrix.prompt }}
+
+            Keep your response focused and concise.
+          allowed_tools: "Read"
+          max_turns: 3
+
+      - name: Validate response contains expected keywords
+        id: validate
+        continue-on-error: true
+        run: |
```

actionlint (1.7.9)

property "response" is not defined in object type {branch_name: string; execution_file: string; github_token: string; structured_output: string}

(expression)